src/pages/api/auth/[...nextauth].js

```diff
@@ -32,7 +32,7 @@ const options = {
   secret: process.env.NEXTAUTH_SECRET,
 
   callbacks: {
-    async jwt({ user, account }) {
+    async jwt({ user, account, token }) {
```

Comment:
token is also now JWT so you should be able to refresh it, etc.

Reply:
sure, lemme handle auth changes in another PR @kilemensi

Reply:
Weren't you the one telling me this PR doesn't need to change @KobbyMmo ? I'm just showing you all these places that need to change bruv!

Reply:
What's the new auth endpoint now? or we are maintaining the old one?

Reply:
Yeah, same one